SPECS/llvm/llvm.spec

```diff
@@ -8,6 +8,7 @@ Distribution:   Mariner
 Group:          Development/Tools
 URL:            https://llvm.org/
 Source0:        https://github.com/llvm/llvm-project/releases/download/llvmorg-%{version}/%{name}-%{version}.src.tar.xz
+Patch1:         llvm-12.0.1-issue-49955-workaround.patch
```

Comment:
Thanks for the update. Now we missed incrementing the `Release` tag and adding a changelog entry. Aside from that, I think we're good to go.

Reply:
Oops when I switched techniques I forgot to pull over that diff. Will fix this now.